vendor/composer/autoload_real_52.php

```diff
@@ -2,7 +2,11 @@
 
 // autoload_real_52.php generated by xrstf/composer-php52
 
+<<<<<<< HEAD
```

Comment:
Conflicts.

Reply:
dang it...why did that not show up for me locally :/